src/CloudNative.CloudEvents/CloudNative.CloudEvents.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>CNCF CloudEvents SDK</Description>
     <LangVersion>8.0</LangVersion>
+    <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
```

Comment:
Removed this now - it was added while I was experimenting with StyleCop, but we can work out how to enforce styles later.